doc/lua_api.txt

```diff
@@ -1865,9 +1865,16 @@ to games.
 
 ### Node-only groups
 
-* `attached_node`: if the node under it is not a walkable block the node will be
-  dropped as an item. If the node is wallmounted the wallmounted direction is
-  checked.
+* `attached_node`: the node is 'attached' to a neighboring node. It checks
+                   whether the node it is attached to is walkable. If it
+                   isn't, the node will drop as an item.
+    * `1`: if the node is wallmounted, node is attached to the wallmounted
+           direction. Otherwise, node is attached to the node below.
```

Comment:
```suggestion
    * `1`: if the node is wallmounted, the node is attached in the wallmounted
           direction. Otherwise, the node is attached to the node below.
```